java/build.gradle

```diff
@@ -122,22 +122,22 @@ services.each { Service svc ->
     tasks.named(svc.id) { dependsOn deployModels, deployServices, deploySerializers, deployWebhookHandlers }
 }
 
-// Test small services generation
+// Test binlookup  generation
 tasks.named('binlookup') {
     doLast {
         assert file("${layout.projectDirectory}/repo/src/main/java/com/adyen/model/binlookup/Amount.java").exists()
         assert file("${layout.projectDirectory}/repo/src/main/java/com/adyen/service/BinLookupApi.java").exists()
         assert !file("${layout.projectDirectory}/repo/src/main/java/com/adyen/service/binlookup/DefaultApi.java").exists()
     }
 }
-// Test services generation
+// Test checkout generation
 tasks.named('checkout') {
     doLast {
         assert file("${layout.projectDirectory}/repo/src/main/java/com/adyen/model/checkout/Amount.java").exists()
         assert file("${layout.projectDirectory}/repo/src/main/java/com/adyen/service/checkout/PaymentsApi.java").exists()
     }
 }
-// Test webhooks generation
+// Test acswebhooks generation
```

Comment:
For maintainability purposes: I'd like to understand two things:

1. Why are we asserting that a specific file should not exist (e.g. because we've moved it)
2. Why are we asserting that a specific should exist? (e.g. because we moved it to a specific output folder)